### PR TITLE
Wizard copy of the Lambda support for .NET

### PR DESCRIPTION
**Pull request by ajjindal**
Thanks @bruno-garcia

### CODE REVIEW

File: src/wizard/dotnet/awslambda.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Install-Package Sentry.AspNetCore -Version {{ packages.version('sentry.dotnet.aspnetcore') }}
+```
+
+```shell {tabTitle:.NET Core CLI}
+dotnet add package Sentry.AspNetCore -v {{ packages.version('sentry.dotnet.aspnetcore') }}
+```
+
+You can combine this integration with a logging library like `log4net`, `NLog`, or `Serilog` to include both request data as well as your logs as breadcrumbs. The logging ingrations also capture events when an error is logged.
+
+We also recommend updating your app/server to run on a more current version of the .NET Framework, such as .NET 4.7.2.
```
Comment (bruno-garcia) [inline src/wizard/dotnet/awslambda.md:24]:
```suggestion
```

File: src/wizard/dotnet/awslambda.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            {
+              o.Dsn = "__PUBLIC_DSN__";
+              // Required in Serverless environments
+              o.FlushOnCompletedRequest = true;
+            })
+            .UseStartup<Startup>();
+    }
+}
+```
+
```
Comment (bruno-garcia) [inline src/wizard/dotnet/awslambda.md:50]: What about a verification step.

Note the missing ` to close the code blocks, needs amending.

```suggestion

### Verification

You can verify your setup by throwing an exception from a function:

```csharp
[Route("api/[controller]")]
public class BadController
{
    [HttpGet]
    public string Get() => throw null;
}
``

And make a request to that lambda:

```sh
curl -X GET -I https://url.of.server.aws/api/bad
``


```

**Review by bruno-garcia — APPROVED** [review]
Awesome stuff!

I left a couple suggestions.
One I did on the docs too: #2713 

The other is a verification step (folks get confused after adding sentry: How do I know if it works?)

File: src/wizard/dotnet/awslambda.md [hunk 3]
```diff
@@ -48,4 +48,24 @@ public class LambdaEntryPoint : Amazon.Lambda.AspNetCoreServer.APIGatewayProxyFu
...
+    [HttpGet]
+    public string Get() => throw null;
+}
+``
+
+And make a request to that lambda:
+
+```sh
+curl -X GET -I https://url.of.server.aws/api/bad
+``
```
Comment (bruno-garcia) [inline src/wizard/dotnet/awslambda.md]: You didn't add the extra ` I pointed out

Comment (ajjindal) [reply src/wizard/dotnet/awslambda.md]: I thought I did, sorry if I missed it. Thanks for taking care of this.

**Pull request merged**